pkg/model/iam/iam_builder.go

```diff
@@ -244,13 +244,12 @@ func (b *PolicyBuilder) BuildAWSPolicyBastion() (*Policy, error) {
 		Version: PolicyDefaultVersion,
 	}
 
-	// Bastion hosts currently don't require any specific permissions.
-	// A trivial permission is granted, because empty policies are not allowed.
-	p.Statement = append(p.Statement, &Statement{
-		Effect:   StatementEffectAllow,
-		Action:   stringorslice.Slice([]string{"ec2:DescribeRegions"}),
-		Resource: resource,
-	})
+	addNodeEC2Policies(p, resource)
```

Comment:
Maybe rename this function to `addBasicEC2Policies` or something? I got a bit confused when bastion called this one.

Reply:
I think I will create a new function called `addBastionEC2Policies` instead.

Reply:
@olemarkus Made a change for this function.